src/identity/interaction/email-password/credentials/AccountVerificationHandler.ts

```diff
@@ -0,0 +1,37 @@
+import assert from 'assert';
+import type { Representation } from '../../../../http/representation/Representation';
+import { MethodNotAllowedHttpError } from '../../../../util/errors/MethodNotAllowedHttpError';
+import { readJsonStream } from '../../../../util/StreamUtil';
+import type { InteractionHandlerInput } from '../../InteractionHandler';
+import { InteractionHandler } from '../../InteractionHandler';
+import type { AccountStore } from '../storage/AccountStore';
+import type { CredentialsHandler } from './CredentialsHandler';
+
+/**
+ * Authenticates a user by the email/password in the JSON body.
+ * Passes the body and the WebID associated with that account to the source handler.
+ */
+export class AccountVerificationHandler extends InteractionHandler {
```

Comment:
It's more like `EmailPasswordAuthorizer` then, right?

Reply:
Indeed, I'll rename. The cause is the current tight coupling between email/password and accounts. This is the class that will definitely disappear after having a more flexible account structure.